src/error.js

```diff
@@ -107,8 +107,7 @@ function reportErrorToServer(message, filename, line, col, error) {
   if (this && this.document) {
     makeBodyVisible(this.document);
   }
-  const mode = getMode();
```

Comment:
@cramforce PTAL. works but unfortunately I don't detect re-aliasing right now, is there a way to get  the type information of a return type? (I'm sure there is I just don't know how to get to it)